contracts/op-finality-gadget/src/state/config.rs

```diff
@@ -11,4 +12,7 @@ pub const IS_ENABLED: Item<bool> = Item::new("is_enabled");
 #[cw_serde]
 pub struct Config {
     pub consumer_id: String,
+    pub consumer_name: Option<String>,
+    pub consumer_description: Option<String>,
```

Comment:
these two fields are unnecessary, we can query them with `consumer_id` from Babylon if need